zfs-status.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+#
+# Queries the status of zpool named by first argument and writes online or failed into
+# the status file at /etc/protonet/system/zfs-status
+#
+zpool list $1 | tail -n +2 | grep ONLINE
```

Comment:
@colszowka when the pool is offline, you'll know because docker dies. Better check with `zpool status`:

```
$ sudo zpool status
  pool: protonet_storage
 state: ONLINE
  scan: scrub repaired 0 in 0h0m with 0 errors on Mon Mar 14 10:29:39 2016
config:

	NAME        STATE     READ WRITE CKSUM
	protonet_storage  ONLINE       0     0     0
	  sda       ONLINE       0     0     0

errors: No known data errors
```